### PR TITLE
feat: add collector delete modal and bind to delete button in CollectorDetailPage

### PR DESCRIPTION
**Pull request by WANZARGEN**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description

SSIA

https://github.com/cloudforet-io/console/assets/26986739/1b9491bc-304c-41e9-b2ce-52087b61a960

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue [hunk 1]
```diff
@@ -174,12 +190,44 @@ const getCollector = async (): Promise<CollectorModel> => {
...
+
+const handleDeleteModalConfirm = async () => {
+    state.deleteModalVisible = true;
+    try {
+        state.deleteLoading = true;
+        await fetchDeleteCollector();
+        state.deleteModalVisible = false;
+        showSuccessMessage(i18n.t('INVENTORY.COLLECTOR.ALT_S_DELETE_COLLECTOR'), '');
+        goBackToMainPage();
+        collectorFormStore.$reset();
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue:224]: praise: 😮

**Review by yuda110 — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue [hunk 2]
```diff
@@ -49,6 +49,13 @@
         <collector-service-accounts-section class="section"
                                             :providers="state.collectorProviders"
         />
+        <p-double-check-modal :visible.sync="state.deleteModalVisible"
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue:52]: Suggestion: Please remove `.sync` 😄

File: apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue [hunk 3]
```diff
@@ -178,12 +194,44 @@ const getCollector = async (): Promise<CollectorModel> => {
     return result;
 };
 
+const fetchDeleteCollector = async () => {
+    // TODO: change to call api
+    await new Promise((resolve, reject) => {
+        setTimeout(() => {
+            reject(makeAPIError('error'));
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue:201]: praise: 👍

**Review by sulmoJ — APPROVED** [review]
LGTM!!!

File: apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue (hunk 2 above)
Comment (WANZARGEN) [reply apps/web/src/services/asset-inventory/collector/collector-detail/CollectorDetailPage.vue:52]: oops! thanx!

**Pull request merged**